### PR TITLE
Add combination of p values to validation pipeline

### PR DESCRIPTION
**Pull request by erickzul**

### CODE REVIEW

File: validation_pipeline.ipynb [hunk 1]
```diff
@@ -25,12 +25,14 @@
         "DEBUG = False #@param {type:\"boolean\"}\n",
```
Comment (rothn) [inline validation_pipeline.ipynb:25]: <pre class="ql-syntax" spellcheck="false">Line #10.    TEST_SET_FILENAME = 'canonical/uc_davis_test_fixed_grouped.csv' #@param
</pre><p>What does "grouped" mean in this context? There was some notion of "grouped" and "ungrouped" as well as per-sample and per-site. How does that fit in here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (rothn) [inline validation_pipeline.ipynb:25]: <pre class="ql-syntax" spellcheck="false">Line #1.    mean_rmse, var_rmse, overall_rmse = evaluation.calculate_rmse(eval_dataset, means_isoscape, vars_isoscape, MEAN_TRUTH_NAME, VAR_TRUTH_NAME, MEAN_PREDICTED_NAME, VAR_PREDICTED_NAME)
</pre><p>Are we averaging the RMSE over all our isoscapes? Is this just for Oxygen? Clarification needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (rothn) [inline validation_pipeline.ipynb:25]: <pre class="ql-syntax" spellcheck="false">Line #1.    mean_iso = means_isoscape
</pre><p>What's the point of this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (rothn) [inline validation_pipeline.ipynb:25]: <pre class="ql-syntax" spellcheck="false">Line #1.    fake_sample = dataset.create_fraudulent_samples(
</pre><p>Is this just for Oxygen or for everything?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (rothn) [inline validation_pipeline.ipynb:25]: <pre class="ql-syntax" spellcheck="false">Line #2.    mean_isoscapes = [
&nbsp; &nbsp; means_isoscape,
</pre><p>This is kind of counterintuitive for me. Can we clarify that those are <em>oxygen</em> means somehow?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

**Review by rothn — CHANGES_REQUESTED** [review]
Thanks for making these changes to further clean up and centralize the code. I need more context on some things here, per my comments.

File: validation_pipeline.ipynb (hunk 1 above)
Comment (erickzul) [reply validation_pipeline.ipynb:25]: It means that it's the test dataset we got from data_ingestion.ipynb by grouping by Code, lat, long.

Comment (erickzul) [reply validation_pipeline.ipynb:25]: Just oxygen. That's the RMSE we measure for the current model we're trying to build.

Comment (erickzul) [reply validation_pipeline.ipynb:25]: Ack, updated.

Comment (erickzul) [reply validation_pipeline.ipynb:25]: Yeah, renamed the variable.

Comment (erickzul) [reply validation_pipeline.ipynb:25]: It's for everything. Updated to reflect this.

Comment (rothn) [reply validation_pipeline.ipynb:25]: That makes sense. If this is meant for end-to-end evaluation though, it should ideally do the same thing for each isoscape, per our [isoscape validation design doc](go/ddf-isoscape-validation)

**Review by rothn — APPROVED** [review]
LGTM in general. If this is meant for end-to-end evaluation, it should do the same thing for each isoscape per our [isoscape validation design doc](https://github.com/tnc-br/ddf-isoscapes/pull/go/ddf-isoscape-validation). That does not necessarily have to be addressed in this PR but should be addressed.

**Pull request merged**